sda-commons-server-mongo-testing/README.md

```diff
@@ -2,25 +2,26 @@
 
 [![javadoc](https://javadoc.io/badge2/org.sdase.commons/sda-commons-server-mongo-testing/javadoc.svg)](https://javadoc.io/doc/org.sdase.commons/sda-commons-server-mongo-testing)
 
-This module provides the [`MongoDbRule`](src/main/java/org/sdase/commons/server/mongo/testing/MongoDbRule.java), 
+This module provides the [`MongoDbRule`](src/main/java/org/sdase/commons/server/mongo/testing/MongoDbRule.java),
 a JUnit test rule that is used to automatically bootstrap a MongoDB instance for integration tests.
 
-This is accomplished using [Flapdoodle embedded MongoDB](https://github.com/flapdoodle-oss/de.flapdoodle.embed.mongo), 
-that downloads and starts MongoDB in a separate process. 
-As an alternative, one can use [FakeMongo Fongo](https://github.com/fakemongo/fongo) to create an 
+This is accomplished using [Flapdoodle embedded MongoDB](https://github.com/flapdoodle-oss/de.flapdoodle.embed.mongo),
+that downloads and starts MongoDB in a separate process.
+As an alternative, one can use [FakeMongo Fongo](https://github.com/fakemongo/fongo) to create an
 in-memory MongoDB database, however it seems to be discontinued and is lacking major features.
 
-The rule can be overridden in a test environment to 
+The rule can be overridden in a test environment to
 [use an existing database](#use-an-existing-database) instead of starting a local MongoDB.
 
 ## Usage
 
 To use the test rule, a dependency to this module has to be added:
 
 ```
-testCompile 'org.sdase.commons:sda-commons-server-mongo-testing:<current-version>'
+testCompile 'org.sdase.commons:sda-commons-server-mongo-testing'
 ```
 
+## JUnit 4
```

Comment:
Shouldn't this start with `###` as it is a sub-heading of "Usage"? I was wondering why this was changed below

Reply:
updated